<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configure</class>
 <widget class="QDialog" name="Configure">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>295</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Configure</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/GalleryDesigner.qrc">
    <normaloff>:/icons/images/configure.png</normaloff>:/icons/images/configure.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_Language">
         <property name="text">
          <string>Prefered language</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_Language"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_AfterProduction">
         <property name="text">
          <string>After producing the gallery</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_AfterProduction">
         <item>
          <property name="text">
           <string>Display the gallery</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Open the output folder</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Do nothing</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_RecentProject">
       <property name="text">
        <string>Open the most recent project when launching EZWebGallery</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_EnablePreview">
       <property name="text">
        <string>Enable galery previewing</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="tristate">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_EmbedAnalytics">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Always enable Google Analytics</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_TrackingId">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Default Google Analytics tracking ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_TrackingId">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/GalleryDesigner.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>checkBox_EmbedAnalytics</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_TrackingId</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_EmbedAnalytics</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_TrackingId</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
